doc/dev/generate.md

```diff
@@ -0,0 +1,96 @@
+<!-- cspell: ignore tspconfig mgmt -->
+
+# Generating Management Plane Crates from TypeSpec
+
+This document complements [Creating a TypeSpec-Based Rust SDK Client](https://github.com/Azure/azure-sdk-for-rust/blob/main/doc/dev/new-typespec-based-client.md). Read that guide first for the canonical, step-by-step walkthrough; use this page to layer on the management plane specifics that keep our Resource Manager crates consistent with the rest of the Azure SDK.
+
+## When to reach for this guide
+
+You are in the right place if all of the following are true:
+
+-   Your TypeSpec project targets Resource Manager APIs (`import "@azure-tools/typespec-azure-resource-manager";`).
+-   You plan to ship the resulting crate under the `azure_mgmt_*` naming pattern in this repository.
+-   You already understand the base TypeSpec → Rust workflow described in `new-typespec-based-client.md` and only need the management-plane adjustments.
+
+## Management-plane prerequisites
+
+In addition to the standard tooling covered in the base guide, keep these points in mind when preparing the TypeSpec inputs:
+
+-   **Emitter options** – `tspconfig.yaml` must emit both the Resource Manager libraries and the Rust emitter. Align the options with other languages so rotation engineers only maintain one config. A minimal block looks like:
+
+    ```yaml
+    emit:
+        - "@azure-tools/typespec-azure-resource-manager"
+        - "@azure-tools/typespec-rust"
+    options:
+        "@azure-tools/typespec-rust":
+            emitter-output-dir: "{output-dir}/{service-dir}/{crate-name}"
+            crate-name: "azure_mgmt_<service>"
+            crate-version: "0.1.0"
+        "@azure-tools/typespec-azure-core":
+            azure-resource-provider: "Microsoft.<ProviderName>"
+    ```
+
+-   **Naming** – Follow the `azure_mgmt_<service>` crate pattern and mirror the service folder layout used by other languages (`sdk/<service>/<crate-name>`). When in doubt, look at the Go SDK’s management plane docs for a matching service and port the structure.
+-   **Spec changes** – Capture the `azure-rest-api-specs` commit SHA (or PR branch commit) that contains the TypeSpec updates; you will reference it from `tsp-location.yaml`.
+
+## Repository setup checklist
+
+The base client guide covers initialization in detail. For management plane, double-check these repo-specific tasks once your service directory exists:
+
+-   Add `sdk/<service>/<crate-name>` to the workspace members in the root `Cargo.toml`, but keep it out of `default-members`.
```

Comment:
It should also be noted that these will go alongside DP crates within the `<service-directory>` e.g., `azure_resourcemanager_keyvault` would go in `sdk/keyvault` with the rest of the `azure_security_keyvault_*` crates.